app/helpers/injection_helper.rb

```diff
@@ -35,13 +35,18 @@ def inject_enterprise_shopfront_list
 
     inject_json_ams(
       "enterprises",
-      Enterprise.activated.visible.select(select_only).includes(address: :state).all,
+      Enterprise.activated.visible.select(select_only).includes(address: [:state, :country]).all,
       Api::EnterpriseShopfrontListSerializer
     )
   end
 
   def inject_enterprise_and_relatives
-    inject_json_ams "enterprises", current_distributor.relatives_including_self.activated.includes(address: :state).all, Api::EnterpriseSerializer, enterprise_injection_data
+    enterprises_and_relatives = current_distributor.relatives_including_self.activated.
+      includes(address: [:state, :country]).all
```

Comment:
style. I prefer to break all lines when one line is broken
```ruby
    enterprises_and_relatives = current_distributor.
         relatives_including_self.
         activated.
         includes(address: [:state, :country]).
         all
```

Reply:
Done :+1: